client/src/components/kvEditor/KVEditorRow.tsx

```diff
@@ -195,6 +202,16 @@ function KVEditorRow({
     <div key={`${name}-${i}`} className={styles.row}>
       {!readOnly ? (
         <>
+          {canDisableRow && (
+            <input
```

Comment:
Let's use the chakra-ui <Checkbox> component here with the correct green coloring.